<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositView</class>
 <widget class="QWidget" name="DepositView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>589</width>
        <height>541</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Deposit amount</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LineDepositAmount"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Deposit term</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LineDepositTerm"/>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Interest rate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LineInterestRate"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Tax rate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LineTaxRate"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Periodicity of payments</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>Every day</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>In the end of the month</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>By the end of the term</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QRadioButton" name="RadioButtonCapitalization">
         <property name="text">
          <string>Capitalization of interest</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="6" column="0">
          <widget class="QPushButton" name="AddReplenishment">
           <property name="text">
            <string>Add replenishment</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <layout class="QVBoxLayout" name="AddRM"/>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="AddWithdrawal">
           <property name="text">
            <string>Add withdrawal</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <layout class="QVBoxLayout" name="AddWD"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Accrued interest</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineAccruedInterest"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Tax amount</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineTaxAmount"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Deposit amount by the end of the term</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineDepositEnd"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="pushButton_Calculate">
           <property name="text">
            <string>Рассчитать</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Reset">
           <property name="text">
            <string>Сбросить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
